x-pack/plugins/lens/public/shared_components/toolbar_popover.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
```

Comment:
I have added this and the LegendSettingsPopover on the shared_components folder. The ToolbarButton component is on a separate folder so if you want to keep this approach for them too, let me know 🙂

Reply:
IMHO toolbar button should also be within `shared_components`, this is an oversight. Feel free to move, but we can also address this later.